docker-compose.yml

```diff
@@ -15,20 +15,25 @@ services:
     extends:
       service: base
     ports:
+      - "5005:5005"
+      - "7155:7155"
       - "9000:9000"
     links:
       - mongo-dev
-    command: >
-      bash -c
-      "sbt -v -d -jvm-debug 5005 \"run
-      -Djava.net.preferIPv4Stack=true
-      -Dhttp.address=0.0.0.0
-      -Dhttp.port=$${PORT}
-      -Dmongodb.db=$${MONGO_DB}
-      -Dmongodb.url=$${MONGO_HOST}
-      -Dmongodb.port=$${MONGO_PORT}
-      -Dmongodb.uri=mongodb://$${MONGO_HOST}:$${MONGO_PORT}/$${MONGO_DB}
-      \""
+    command:
```

Comment:
Couldn't we just use the `fossildb` docker container here instead of `./sbt`?

Reply:
Yes, that's the nicer option. I'll change that.